src/components/procedure/_platform-add-repo.md

```diff
@@ -13,5 +13,15 @@ To add a repository from GitHub, GitLab, or BitBucket, follow these steps:
 4. Click **Scan new project**, and then click **Run scan in CI**.
 5. Select which CI provider for Semgrep to integrate with.
 6. Follow the instructions displayed on Semgrep Cloud Platform page for your particular CI provider.
+7. Optional: For GitHub Actions users, if you have given Semgrep access to more than 1 repository, you can bulk add many repositories.
 
 You have now added a repository to Semgrep Cloud Platform. A scan begins automatically after adding a new repository.
+
+:::caution
+For **GitHub** users: This method of adding a repository commits a GitHub Actions workflow file directly into your trunk or default branch, such as `main` or `develop`. **Branch protection** can prevent Semgrep from onboarding your repositories, whether in bulk or one-by-one, through this method.
+
+If you encounter issues with adding repositories, such as being unable to commit the Semgrep workflow file, change your [branch protection](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches) settings temporarily while you add repositories to Semgrep.
```

Comment:
I don't think most customers will actually be willing/able to modify their branch protection settings.  I'd also suggest that they introduce the workflow file through a PR and then merge it into the repo.

As the feature bakes - I'd also encourage linking somewhere in here to auto-scan as an alternative.

Reply:
> I'd also suggest that they introduce the workflow file through a PR and then merge it into the repo.

Thanks for the reply. Just confirming that you're suggesting, IF the user CAN'T change their branch protection: 
1. The user has to do it manually (copy paste the yml file, make a PR).
2. The user has to do this for every repo.

Is that correct?

Reply:
Yup that's exactly right. 

Regarding the "do this for every repo" - it might be helpful to caveat if they are looking to deploy to many repos where this would be cumbersome - they might consider some of the approaches outlined by github here:

https://github.blog/2023-04-05-building-organization-wide-governance-and-re-use-for-ci-cd-and-automation-with-github-actions/

Reply:
we could also link to our required/reusable workflows KBs!